gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/alttext/AltTextPage.kt

```diff
@@ -175,7 +185,26 @@ internal fun AltTextPage(
                         textStyle = LocalTextStyle.current.copy(color = MaterialTheme.colorScheme.onSurface),
                         modifier = Modifier
                             .fillMaxWidth()
-                            .padding(start = 16.dp, top = 8.dp, bottom = 8.dp),
+                            .fillMaxHeight(),
+                        decorationBox = { innerTextField ->
+                            Column(
```

Comment:
This is not ideal as the counter is technically below the input field, not inside it, but there's no easy way to handle that so that the text doesn't overlap with the counter.